onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.h

```diff
@@ -6,11 +6,14 @@
 #include <unordered_set>
 
 #include "core/common/inlined_containers.h"
+#include "core/common/inlined_containers_fwd.h"
```

Comment:
"inlined_containers.h" includes "inlined_containers_fwd.h", so don't need to include both
```suggestion
#include "core/common/inlined_containers.h"
```

Reply:
Will fix it in the next PR.